--- sunxi-mali/include/GLES2/gl2ext.h	2020-10-15 18:21:37.496914633 +0200
+++ sunxi-mali.patch/include/GLES2/gl2ext.h	2020-10-15 22:56:59.962977786 +0200
@@ -79,6 +79,8 @@
 #define GL_DEPTH24_STENCIL8_OES                                 0x88F0
 #endif
 
+#define GL_BGRA_EXT 0x80E1
+
 /* GL_OES_rgb8_rgba8 */
 #ifndef GL_OES_rgb8_rgba8
 #define GL_RGB8_OES                                             0x8051
